gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,35 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed mostly through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create resources through Anypoint Platform _with the exception_ of Configuration and Service resources. 
+
+In addition, resource creation through `kubectl apply` is restricted to the following resource types:
+
+* Configuration
+* Secret
+* Extension
+* Service
+* PolicyBinding of types `tls`, `tls-inbound`, and `tls-outbound`
+
+// end::k8s-connected-intro[]
+//
+// tag::k8s-local-intro[]
+In Local Mode, Flex Gateway typically acts as an ingress controller that manages external access to your Kubernetes cluster. When using this mode, you must know what resources to create and apply, and use `kubectl apply` to deploy Kubernetes targets and resources such as APIs, policies, or Flex Gateway-related resources to your gateway. 
```

Comment:
this one qualifies that we're talking about using the command for _Kubernetes_ targets and resources